src/Workspaces/Core/Portable/Storage/SQLite/Interop/NativeMethods.cs

```diff
@@ -138,6 +139,16 @@ public static Result sqlite3_bind_text(SafeSqliteStatementHandle stmt, int index
             return (Result)raw.sqlite3_bind_text(stmt.DangerousGetWrapper(), index, val);
         }
 
+        /// <summary>
+        /// <paramref name="val"><see cref="Encoding.UTF8"/> encoded bytes of a text value.  Span
+        /// does not need to be zero-terminated.</paramref>
```

Comment:
```suggestion
        /// <paramref name="val"><see cref="Encoding.UTF8"/> encoded bytes of a text value.  Span
        /// should not be NUL-terminated.</paramref>
```

Reply:
📝 The change "does not need to be" &rarr; "should not be" is important. The original text could lead to misuse of the API where NUL characters are part of the text itself.

Reply:
wfm.